src/vcpkg-test/platform-expression.cpp

```diff
@@ -152,6 +152,48 @@ TEST_CASE ("platform-expression-and", "[platform-expression]")
     }));
 }
 
+TEST_CASE ("platform-expression-and-alternate", "[platform-expression]")
+{
+    auto m_expr = parse_expr("!windows and !arm");
+    REQUIRE(m_expr);
+    auto& expr = *m_expr.get();
+
+    CHECK_FALSE(expr.evaluate({{"VCPKG_CMAKE_SYSTEM_NAME", ""}}));
+    CHECK_FALSE(expr.evaluate({{"VCPKG_CMAKE_SYSTEM_NAME", "WindowsStore"}}));
+    CHECK(expr.evaluate({{"VCPKG_CMAKE_SYSTEM_NAME", "Linux"}}));
+    CHECK_FALSE(expr.evaluate({
+        {"VCPKG_CMAKE_SYSTEM_NAME", "Linux"},
+        {"VCPKG_TARGET_ARCHITECTURE", "arm"},
+    }));
+}
+
+TEST_CASE ("platform-expression-and-multiple", "[platform-expression]")
+{
+    auto m_expr = parse_expr("!windows & !arm & !x86");
```

Comment:
I don't see any tests for "minimum whitespace" examples:
`windows&arm`
`windows,arm&!linux`
etc.
These are supported today (though `format-manifest` inserts spaces)

Reply:
Good catch. Added minimum whitespace test.